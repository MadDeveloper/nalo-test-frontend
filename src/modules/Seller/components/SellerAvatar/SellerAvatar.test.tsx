import renderer from "react-test-renderer"
import { SellerAvatar } from "."

const seller = {
  name: "Richard Noga",
  id: 2323,
  isVerified: true,
  avatar: "/assets/images/sellers/richard-noga.jpeg",
}

describe("Seller module > SellerAvatar", () => {
  it("should match snapshot", () => {
    const tree = renderer.create(<SellerAvatar seller={seller} />).toJSON()

    expect(tree).toMatchSnapshot()
  })

  it("should match snapshot with wallet amount", () => {
    const tree = renderer
      .create(<SellerAvatar seller={seller} showWalletAmount />)
      .toJSON()

    expect(tree).toMatchSnapshot()
  })
})
