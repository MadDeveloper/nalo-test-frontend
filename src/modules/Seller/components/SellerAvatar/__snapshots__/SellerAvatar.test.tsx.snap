// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Seller module > SellerAvatar should match snapshot 1`] = `
<div
  className="root"
>
  <div
    className="root"
    style={
      Object {
        "backgroundImage": "url(/assets/images/sellers/richard-noga.jpeg)",
      }
    }
  >
    <svg
      aria-hidden="true"
      className="root verified"
      fill="currentColor"
      viewBox="0 0 24 24"
      xmlns="http://www.w3.org/2000/svg"
    >
      <path
        clipRule="evenodd"
        d="M2.25 12c0-5.385 4.365-9.75 9.75-9.75s9.75 4.365 9.75 9.75-4.365 9.75-9.75 9.75S2.25 17.385 2.25 12zm13.36-1.814a.75.75 0 10-1.22-.872l-3.236 4.53L9.53 12.22a.75.75 0 00-1.06 1.06l2.25 2.25a.75.75 0 001.14-.094l3.75-5.25z"
        fillRule="evenodd"
      />
    </svg>
  </div>
  <div>
    <span
      className="root bold name"
    >
      Richard Noga
    </span>
    <br />
  </div>
</div>
`;

exports[`Seller module > SellerAvatar should match snapshot with wallet amount 1`] = `
<div
  className="root"
>
  <div
    className="root"
    style={
      Object {
        "backgroundImage": "url(/assets/images/sellers/richard-noga.jpeg)",
      }
    }
  >
    <svg
      aria-hidden="true"
      className="root verified"
      fill="currentColor"
      viewBox="0 0 24 24"
      xmlns="http://www.w3.org/2000/svg"
    >
      <path
        clipRule="evenodd"
        d="M2.25 12c0-5.385 4.365-9.75 9.75-9.75s9.75 4.365 9.75 9.75-4.365 9.75-9.75 9.75S2.25 17.385 2.25 12zm13.36-1.814a.75.75 0 10-1.22-.872l-3.236 4.53L9.53 12.22a.75.75 0 00-1.06 1.06l2.25 2.25a.75.75 0 001.14-.094l3.75-5.25z"
        fillRule="evenodd"
      />
    </svg>
  </div>
  <div>
    <span
      className="root bold name"
    >
      Richard Noga
    </span>
    <br />
    <span
      className="root secondary walletAmount"
    >
      0.00
       ETH
    </span>
  </div>
</div>
`;
